play-services-api/src/main/aidl/com/google/android/gms/panorama/internal/IPanoramaService.aidl

```diff
@@ -0,0 +1,13 @@
+/*
+ * SPDX-FileCopyrightText: 2023 microG Project Team
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.google.android.gms.panorama.internal;
+
+import com.google.android.gms.panorama.internal.IPanoramaCallbacks;
+import android.os.Bundle;
+import android.net.Uri;
+
+interface IPanoramaService {
+    void loadPanoramaInfoAndGrantAccess(IPanoramaCallbacks callback, in Uri uri, in Bundle bundle, boolean needGrantReadUriPermissions) = 0;
```

Comment:
Should be called `loadPanormaInfo`. To grant access or not is handled by the client and informed via the last parameter bool.